src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const basePrice = 40;
+  const numberOfDays = days;
```

Comment:
The variable `numberOfDays` is redundant since it's a direct assignment of the `days` parameter. You can use the `days` parameter directly in your calculations.